e2e/terraform/shared/config/provision-client.sh

```diff
@@ -52,9 +52,42 @@ wget -q -O - \
      https://github.com/containernetworking/plugins/releases/download/v0.8.6/cni-plugins-linux-amd64-v0.8.6.tgz \
     | sudo tar -C /opt/cni/bin -xz
 
+# enable varlink socket (not included in ubuntu package)
+cat > /etc/systemd/system/io.podman.service << EOF
+[Unit]
+Description=Podman Remote API Service
+Requires=io.podman.socket
+After=io.podman.socket
+Documentation=man:podman-varlink(1)
```

Comment:
Probably worth including the link to https://podman.io/blogs/2019/01/16/podman-varlink.html here b/c the person reading isn't super-likely to have the manpage handy.